src/lv_api_map.h

```diff
@@ -45,13 +45,11 @@ static inline LV_ATTRIBUTE_TIMER_HANDLER uint32_t lv_task_handler(void)
   * DEPRECATED FUNCTIONS
   **********************/
 
-/* function is inline in lv_obj_tree.h, otherwise Micropython will not build
-inline uint32_t lv_obj_get_child_id(const struct _lv_obj_t* obj)
+static inline uint32_t lv_obj_get_child_id(const struct _lv_obj_t* obj)
 {
     LV_LOG_USER("lv_obj_get_child_id(obj) is deprecated, please use lv_obj_get_index(obj).");
```

Comment:
`LV_LOG_USER` is for logs sent by the user. I'd use `LV_LOG_WARN` here.

Reply:
done